integration/app/test-demo/src/app.tsx

```diff
@@ -19,6 +22,7 @@ const App: React.FC = () => (
 export const routes = {
   HOME: '/',
   ROSTER_TEST: 'roster-test',
+  VIDEO_FILTER_TEST: 'video-filter',
```

Comment:
Could we remove the `_TEST` suffix, I feel like it's kinda redundant.
If you agree, I will update the route of Roster Test App later as well:
```
ROSTER: 'roster'
```

Reply:
I dont think its redundant, we're just being explicit. Better to be more explicit, especially since we have a component called <Roster/>